### DISCUSSION TITLE
README review

### DISCUSSION THREAD

**Discussion by giorgionicoli** [Textualize/textual#42] [Show and tell]
Hi Will and Textual community.

This is Giorgio and I have been reviewing the README file :) I'll post here my comments, with screenshot if you find it helpful.

Here maybe the problem is that my English is not good enough, but I didn't get the meaning of this sentence. Maybe changing "that" to "than" works, but I still would not get it :S

![image](https://user-images.githubusercontent.com/75094347/128599365-42a4d5cf-06f5-4958-a2e0-16090ccc017e.png)

Just a small detail anyway. I'll post more here if I find something :)

**Comment by giorgionicoli** [comment]
Here I was unsure exactly how the change to the background should work. Is there a convention where you set it to a string formatted in a certain way? Or is this string referring to a method that gets called by eval or in some other method?

I would suggest to add a line to clarify whether there's some convention/assumption made here :)

![image](https://user-images.githubusercontent.com/75094347/128599520-9be40a80-8b22-49e2-8e92-a58114389ef3.png)

**Reply by giorgionicoli** [reply]
![image](https://user-images.githubusercontent.com/75094347/128600422-a2ca5765-4e26-4164-a6e4-bc63c377bd94.png)

**Reply by giorgionicoli** [reply]
![image](https://user-images.githubusercontent.com/75094347/128601580-a6c8dc4f-d9e3-4ad6-b06e-2744e5cd74d8.png)

**Reply by giorgionicoli** [reply]
![image](https://user-images.githubusercontent.com/75094347/128601736-83a34485-8367-4e9d-b41d-e0ec5a00e531.png)

**Comment by NoblySP** [comment]
Sorry about the fact that I submitted a PR before seeing that Giorgio was already discussing about the README. Should I do something about the PR (like deleting it if that's an option) or is it fine if I leave it alone?

**Reply by giorgionicoli** [reply]
I saw it, thank you for helping! I will continue with typos from where you left-off then. @willmcgugan would you hate if I would add a second PR later that covers the part not modified by @NoblySP ?

**Comment by willmcgugan** [comment]
@qJakc Curses is an old library for building TUIs. I might link to the Wikipedia page there.

The background string is a Rich [style](https://rich.readthedocs.io/en/latest/style.html). I should link to the Rich docs there so the reader has context.

Not sure about a sample log file yet. The format is subject to change and it will quickly get out of date.

Good point about the units for timers.

Re animate a value, that's not very well worded. It basically means change one value to another value in small increments over time.

Some good points there! Feel free to PR those changes.

BTW images aren't great for discussions. If you stick to markdown it makes it easier to quote your comments

**Reply by giorgionicoli** [reply]
I'm happy to help! :)
Yeah, after doing this I also realised that screenshots are not the best. I wanted to quickly give context about the comment and that's what I thought of. Next time I will just use markdown as you suggest, paste the section of the readme and highlight changes.
